src/basic/constants.h

```diff
@@ -73,6 +73,12 @@
         "/usr/local/lib/" n,                    \
         "/usr/lib/" n
 
+#define CONF_PATHS_RUN_FIRST(n)                 \
```

Comment:
please add a comment here, explaining when to use this, because this doesn't appear like something people should typically use. it's an exception afaiu